packages/@coorpacademy-components/src/organism/mooc-header/index.js

```diff
@@ -269,9 +269,12 @@ class MoocHeader extends React.Component {
       'data-name': `item-more-${item.title}`,
       customStyle: item.selected
         ? {
-            color: primaryColor
+            color: primaryColor,
+            padding: '0 16px'
+          }
+        : {
+            padding: '0 16px'
```

Comment:
padding: '0 16px' est toujours là peut importe si selected ou pas
```suggestion
customStyle: {
  padding: '0 16px',
  ...(item.selected && { color: primaryColor })
}
```

Reply:
bien vu